cinema_service/settings.py

```
@@ -1,28 +1,17 @@
-"""
-Django settings for cinema_service project.
+import os
 
-Generated by 'django-admin startproject' using Django 4.0.4.
-
-For more information on this file, see
-https://docs.djangoproject.com/en/4.0/topics/settings/
-
-For the full list of settings and their values, see
-https://docs.djangoproject.com/en/4.0/ref/settings/
-"""
 from datetime import timedelta
 from pathlib import Path
 
+from dotenv import load_dotenv
+
+load_dotenv()
+
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
 
-# Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
-
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = os.environ["SECRET_KEY"]
```

Using os.environ[...] for SECRET_KEY will raise a KeyError if SECRET_KEY is not provided at import time. In containerized environments it's better to either: (a) use `SECRET_KEY = os.environ.get('SECRET_KEY') or raise ImproperlyConfigured('SECRET_KEY not set')`, or (b) provide a safe default for local development but fail loudly for production. This prevents obscure tracebacks during startup and gives a clearer error message when the environment is misconfigured. See settings module for context.